external/storm-solr/src/main/java/org/apache/storm/solr/bolt/AbstractSolrBolt.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.storm.solr.bolt;
```

Comment:
Do we need this class since we only have SolrUpdateBolt or do you have plans on adding any other bolts extending this.